src/apps/HALOEXCHANGE-Cuda.cpp

```diff
@@ -163,6 +164,86 @@ void HALOEXCHANGE::runCudaVariant(VariantID vid)
 
     HALOEXCHANGE_DATA_TEARDOWN_CUDA;
 
+  } else if ( vid == RAJA_WORKGROUP_CUDA ) {
```

Comment:
Is there a reason to add a new variant like this? Can't it just be RAJA_CUDA?

Reply:
You could do that. The base cuda version is there to compare against.
I was trying to have a kind of before and after with both Raja variants.

Reply:
All its really showing between the BASE_CUDA and RAJA_CUDA variants currently is any overhead added by RAJA.

Reply:
what are the two RAJA variants?

Reply:
The RAJA_CUDA version launches one kernel per variable per message in the same manner as the BASE_CUDA variant.
The RAJA_WORKGROUP_CUDA variant uses kernel fusion to launch only one kernel.

Reply:
OK. Got it. Thanks.